components/Home/MainBanner.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import Link from "next/link";
+import FsLightbox from "fslightbox-react";
+import Lottie from "lottie-react";
+import animatedData from "../assets/animated.json"; // Adjust the path as needed
+
+export const BOOKING_LINK = "https://cal.com/shayan-salehis-btb8mk/30min";
+const styles = {
+  bannerLarge: {
+    left: "10%",
+    position: "relative",
+    width: "350px",
+    height: "350px",
+    borderRadius: "100px",
+    overflow: "hidden",
+  },
+  bannerSmall: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    width: "250px",
+    height: "250px",
+    borderRadius: "100px",
+    overflow: "hidden",
+    margin: "auto", // Center the entire div horizontally
+  },
+};
+const AnimatedHeroImage = () => {
```

Comment:
add new lines before and after root level constants, styles, functions or functional components